ivy_tests/test_ivy/test_functional/test_nn/test_layers.py

```diff
@@ -237,6 +237,32 @@ def _mha_helper(draw):
     )
 
 
+@st.composite
+def _nms_helper(draw):
+    img_width = draw(st.integers(250, 500))
```

Comment:
I think we can have a larger bound than 250-500, are there any memory issues for images larger than `500x500`? Same applies to `num_boxes` and the hardcoding of every box to be `20x20`.

Reply:
Yes earlier I kept the bound low for faster testing speed, but i guess that can be changed now. I'm not sure what you mean by `Same applies to num_boxes and the hardcoding of every box to be 20x20.` Are you suggesting to hardcode each box to size `20x20`? I would say it is better to keep a variable size for increased coverage in testing, because object detection models wouldn't predict exactly same size of boxes. Happy to know your opinion @vedpatwardhan

Reply:
Exactly, I was referring to line [`247`](https://github.com/unifyai/ivy/pull/23668/files#diff-c095e30892b77173c50758b5b41a1146ccfd61049e0ba662317b2adddf7e303eR247) where we seem to be hardcoding the box to be `20x20` while it would be great to allow different box sizes 😄

Reply:
Oh no, not really. So basically if `img_width=500`, `x1` can be anything in range `[0,480]` and width of `bbox` can be anything between `[5,500-x1]`. Thus implying `x2=x1+w`, which can be anything from `[5,500]`. `20` is just a random number so that the boxes are well within the range, obviously we can set it to something more smaller like `10`, but I thought `20` is good enough too. Happy to know your thoughts @vedpatwardhan 
```
        x1 = draw(st.integers(0, img_width - 20))
        w = draw(st.integers(5, img_width - x1))
```

Reply:
oh sorry, my bad! Yeah then we're good to go with our data generation. One final change, let's please also update all the `requirement_mappings.json` files and add `torchvision` to the list for `torch`.
Feel free to resolve this conversation and get the PR merged once this is done without a review, thanks 😄

Reply:
Done, thanks!